### ISSUE TITLE
[SerpApi Account] Add Support for Updating Phone Number

### ISSUE THREAD

**Issue by aliayar** [serpapi/public-roadmap#1023]
One of our users asked if updating phone number is possible. Currently, it is not. We can add this to "Edit Profile" page.

![phone](https://github.com/serpapi/public-roadmap/assets/64849049/a6dbe4a0-cfc7-470e-ad2c-e4a45ee377cd)

[Intercom](https://app.intercom.com/a/inbox/yzck75jr/inbox/shared/all/conversation/102910000045794?view=List)

**Comment by schaferyan** [comment]
This is potentially a great idea. Users sometimes can't complete the phone verification if they've already used their phone number to verify an account. Allowing users to edit their phone numbers could be a good workaround for that.

The only potential downside is that users might think we are using the phone number for something besides verification. Perhaps we could include text clarifying that?

**Comment by aliayar** [comment]
I agree with Ryan about explaining our reasoning why we need their phone numbers.

Some of the users are really uncomfortable with sharing their phone numbers.
